pkgs/development/libraries/libvpx/default.nix

```diff
@@ -65,7 +65,15 @@ stdenv.mkDerivation rec {
     sha256 = "sha256-EZP33U10fchyqy7Jr26vHgUUfWR6xtG3fcMWUII0m9w=";
   };
 
-  postPatch = "patchShebangs .";
+  postPatch = ''
+    patchShebangs .
```

Comment:
Please be more specific which files need to be patched.

Reply:
Used some wildcards, but hope it's now specific enough.